packages/experimental/mosaic/src/components/Card/util.tsx

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React, { FC, ReactNode } from 'react';
+
+import { mx } from '@dxos/react-components';
+
+// TODO(burdon): Factor out as hook.
```

Comment:
Why would this need to be a hook? It’s okay if they’re just string constants IMO.